src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -181,6 +182,29 @@ public function setValue(&$objectOrArray, $propertyPath, $value)
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getValues(array $array, $propertyPath): array
+    {
+        $values = [];
+        foreach ($array as $item) {
+            try {
+                $value = $this->getValue($item, $propertyPath);
+
+                if (null === $value) {
+                    continue;
+                }
+
+                $values[] = $value;
+            } catch (ExceptionInterface $e) {
```

Comment:
IMHO, catching exceptions silently here is a no go. `getValues` needs to be coherent with `getValue` and with the whole class. Handling those exceptions is userland responsibility.